packages/atomic/src/components/facets-v1/atomic-timeframe/atomic-timeframe.tsx

```diff
@@ -0,0 +1,43 @@
+import {Component, Prop} from '@stencil/core';
+import {RelativeDatePeriod, RelativeDateUnit} from '@coveo/headless';
+import {Timeframe} from './timeframe';
+
+/**
+ * The `atomic-timeframe` component defines a timeframe of an `atomic-timeframe-facet`, and therefore must be defined within an `atomic-timeframe-facet` component.
+ *
+ * A timeframe is a span of time from now to a specific time in the past.
+ */
+@Component({
+  tag: 'atomic-timeframe',
+  shadow: false,
+})
+export class AtomicTimeframe implements Timeframe {
+  /**
+   * The relative period of time.
+   */
+  @Prop() public period: RelativeDatePeriod = 'past';
+
+  /**
+   * The unit used to define:
+   * - the start date of the timeframe, if the period is `past`
+   * - the end date of the timeframe, if the period is `future`
+   */
+  @Prop() public unit!: RelativeDateUnit;
+
+  /**
+   * The amount of units from which to count.
+   *
+   * E.g., 10 days, 1 year, etc.
+   */
+  @Prop() public amount!: number;
+
+  /**
+   * If `true`, the date will be returned unshifted. If `false`, the date will be adjusted to UTC time.
+   */
+  @Prop() public useLocalTime = true;
```

Comment:
I forget how this option is supposed to work. I see that if `true`, the date is not shifted to UTC.

Q: Are dates in the index normalized to utc before being stored?

If so, then I think we should always shift, and not make it an option to do otherwise?

My thinking is if we do not shift, and if a user in Montreal selects `last day - now`, where now is 1pm Montreal/6pm UTC, if the search api interprets the 1pm as UTC, then the returned results will actually be until 8am Montreal/1pm UTC, meaning the most recent result the user would see is 5 hours old?

cc: @dreisch-coveo , @suhshinhwang would appreciate your input on this,

Reply:
I agree with Sami, I don't think we should make this an option. When I select "last week" I expect to see the results from last week (last 168 hours), it would be weird to see some slightly older results and not see some newer ones because I'm in a different time zone than the results were indexed.

This option also feels confusing to use, since as a developer using the UI I wouldn't immediately know the difference in the results I'll see when this is `true` vs `false`.

Reply:
Good question

Reply:
> if the search api interprets the 1pm as UTC,

Why would it do that ? It just uses the date the client is requesting. 

An ISO date already contains the information about the timezone you are interested in. 
The API does not have to interpret anything.

Reply:
The format sent to the search-api is `YYYY/MM/DD@HH:mm:ss` which I believe is interpreted as UTC.
If the api can accept ISO, adjusting headless to send that format could be another solution.

However, in both cases, the `useLocalTime` option seems unnecessary and confusing,

Reply:
Why was `useLocalTime` used in Headless in the first place? If we deem it unnecessary, should we deprecate it in the rest of the codebase?